bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -350,11 +357,24 @@ public static boolean isDiscardErrorClass(@NonNull String name) {
     @SuppressWarnings("unused")
     public static void deliverReport(@Nullable byte[] releaseStageBytes,
                                      @NonNull byte[] payloadBytes,
+                                     @Nullable byte[] staticDataBytes,
```

Comment:
This method gets called from C, not Java.

Reply:
Yup. I'm referring to maybe using `json_parse_file` in `json_writer.c` and merging the trees there.

Reply:
I worry about differing JSON implementations in how they deal with map collisions. If we merge in Java land we have the established semantics for how the Map interfaces work, whereas with JSON codecs it's anyone's guess what will overwrite what in case of collision.